docs/website/docs/plus/features/projects.md

```diff
@@ -241,7 +324,30 @@ pipelines:
     destination: filesystem
     dataset_name: bronze
 ```
+The current module provides access to various parts of your active dlt+ project:
```

Comment:
You need to introduce how to get the "current" module first - otherwise how do people know how to import it?